src/class-parsely.php

```diff
@@ -1107,4 +1114,23 @@ public function get_api_key(): string {
 
 		return $this->api_key_is_set() ? $options['apikey'] : '';
 	}
+
+	/**
+	 * Get the default category for posts.
+	 *
+	 * @since 3.2.0
+	 *
+	 * @return WP_Term|null The category, or null if a category couldn't be retrieved.
+	 */
+	public function get_default_category(): ?WP_Term {
+		$result      = null;
+		$category_id = get_option( 'default_category' );
```

Comment:
The [usage](https://github.com/wordpress/wordpress/search?q=default_category) of `get_option( 'default_category' )` in WP core doesn't have any fallback option, so if that is understood to always be an integer value, then:

```php
public function get_default_category(): WP_Term {
	return get_term( get_option( 'default_category' ), 'category' );
}
```

should be all you need for this function, which as noted before, could just as easily be used as a one-liner in the places where it's actually needed, rather than define and test a new method.